app/routes.py

```diff
@@ -9,11 +9,18 @@
 def index():
     return "Hello, World!", 200
 
+@JOBS.route('/ping')
+def ping():
+    return "pong", 200
+
 
 @JOBS.route('/submit/<string:song_name>')
 def submit(song_name):
     jobs = job_service.setup_jobs(song_name)
-    return jsonify(job_ids=jobs)
+    if jobs:
+        return jsonify(job_ids=jobs)
+    else:
+        return {"status", 'Unable to find video stream!'}, 424
```

Comment:
Correct code, but I think the correct message should be "Unable to begin transcription due to missing video stream." or something like that.